ci-operator/step-registry/ipi/conf/vsphere/dns/ipi-conf-vsphere-dns-commands.sh

```diff
@@ -22,10 +22,17 @@ then
     export PATH="${HOME}/.local/bin:${PATH}"
     if ! command -v pip3 &> /dev/null
     then
-        echo "$(date -u --rfc-3339=seconds) - No pip available exiting..."
-        exit 1
+        if [ "$(python -c 'import sys;print(sys.version_info.major)')" -eq 2 ]
+        then
+          easy_install --user 'pip<21' 
+          pip install --user awscli
+        else
+          echo "$(date -u --rfc-3339=seconds) - No pip available exiting..."
+          exit 1
+        fi
+    else
+      pip3 install --user awscli
```

Comment:
Keep the indentation consistent.
```suggestion
        pip3 install --user awscli
```